code/_onclick/item_attack.dm

```diff
@@ -160,6 +160,21 @@
 
 /// The equivalent of [/obj/item/proc/attack] but for alternate attacks, AKA right clicking
 /obj/item/proc/attack_alt(mob/living/victim, mob/living/user, params)
+	if(sharpness == SHARP_POINTY | SHARP_EDGED)
```

Comment:
I would much rather that this is done with a component, if only to reduce the amount of vars on items

Reply:
If I get a hang on it sure I could try that

Reply:
You also shouldn't be comparing bitflags with `==` like this.

Reply:
@Jared-Fogle i've been working with him on this, this code should be a bit outdated